packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1186,7 +1186,7 @@ contract VotingV2 is
         unchecked { return x + 1; }
     }
 
-    function _getIdentifierWhitelist() private view returns (IdentifierWhitelistInterface supportedIdentifiers) {
+    function _getIdentifierWhitelist() private view returns (IdentifierWhitelistInterface) {
```

Comment:
Is the getIdentifierWhitelist issue in GovernorV2 addressed in this PR?

Reply:
was not, good catch and fixed.